src/utils/zulipVersion.js

```diff
@@ -26,32 +27,32 @@ type VersionElements = {
 export class ZulipVersion {
   _raw: string;
   _comparisonArray: number[];
-  _loggingArray: number[];
+  _elements: VersionElements;
 
   constructor(raw: string) {
     this._raw = raw;
-    this._comparisonArray = this._getComparisonArray(raw);
-    this._loggingArray = this._getLoggingArray(raw);
+    this._comparisonArray = this._getComparisonArray();
+    this._elements = this._getElements();
```

Comment:
Hmm. I like moving these methods away from being own-properties on the instances. But one thing I don't like about this change as it stands is that it causes these two functions to have implicit dependencies on the order in which we initialize things.

Looking at them again, I think what they really want to be is pure functions. Probably as static methods on this class, since they're closely tied to the details of its implementation. (The alternative would be as module-level, unexported functions.)